src/Interpreters/QueryLog.cpp

```diff
@@ -54,6 +55,11 @@ Block QueryLogElement::createBlock()
 
         {std::make_shared<DataTypeString>(),                                  "current_database"},
         {std::make_shared<DataTypeString>(),                                  "query"},
+        {std::make_shared<DataTypeUInt64>(),                                  "normalized_query_hash"},
+        {std::make_shared<DataTypeLowCardinality>(std::make_shared<DataTypeString>()), "query_kind"},
+        {std::make_shared<DataTypeArray>(std::make_shared<DataTypeString>()), "databases"},
```

Comment:
I'd prefer using LowCardinality for database, table, columns...

Reply:
We already have DataTypeMap, is that better to make ` ProfileEvents` to be the type of  `DataTypeMap` ?

Reply:
Done